modules/git/remote.go

```diff
@@ -24,11 +25,20 @@ func GetRemoteAddress(ctx context.Context, repoPath, remoteName string) (*url.UR
 
 	result, _, err := cmd.RunStdString(&RunOpts{Dir: repoPath})
 	if err != nil {
-		return nil, err
+		return "", err
 	}
 
 	if len(result) > 0 {
 		result = result[:len(result)-1]
 	}
-	return url.Parse(result)
+	return result, nil
+}
+
+// GetRemoteAddress returns the url of a specific remote of the repository.
+func GetRemoteAddress(ctx context.Context, repoPath, remoteName string) (*giturl.GitURL, error) {
```

Comment:
This is returning an URL (albeit a new GitURL) so it should be GetRemoteURL - the string returning thing should be the GetRemoteAddress